src/Gallery.js

```diff
@@ -265,9 +267,12 @@ export default class Gallery extends PureComponent {
             gestureResponder = {};
         }
 
+        const flatListProps = Object.assign(DEFAULT_FLAT_LIST_PROPS, this.props.flatListProps)
```

Comment:
Much better ! Could you use the spread operator here to keep the codebase more consistant ?